test/builder_test.js

```diff
@@ -72,6 +72,25 @@ describe('Builder', function() {
   })
 
   describe('broccoli-plugin nodes (nodeType: "transform")', function() {
+    it('does not create a cachePath when opt-ed out', function() {
```

Comment:
Note: move this back into the multidep require block below with a `if (semver.gt(version, '1.3.0')) {}` guard when https://github.com/broccolijs/broccoli-plugin/pull/19 lands.